managed-databases/postgresql-and-mysql/reference-content/security-and-reliability.mdx

```diff
@@ -0,0 +1,223 @@
+
+### Metrics and logs
+
+Database default logs and metrics are stored inside **Cockpit**.
+
+This includes:
+
+  - **Metrics**: CPU, memory, disk usage, replication lag, and number of connections. Metrics are refreshed every minute.
+  - **Logs**:
+      - PostgreSQL logs at a default level `log_error_verbosity="VERBOSE"` ([see PostgreSQL documentation](https://www.postgresql.org/docs/current/runtime-config-logging.html#GUC-LOG-ERROR-VERBOSITY)). Every request longer than 5 seconds is also logged.
+        <Message type="note">
+          The `log_min_duration_statement` parameter is set at `5000` and cannot be changed.
+        </Message>
+
+        <Message type="tip">
+          For more granular log details, you can activate the [pgaudit extension](/managed-databases/postgresql-and-mysql/api-cli/pg-audit/).
+        </Message>
+      - MySQL logs at a default level `log_error_verbosity=2` ([see MySQL documentation](https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_log_error_verbosity)). Logging slow queries is not activated by default, but you can activate it using `slow_query_log` and `long_query_time`. 
+
+<Message type="important">
+  Any changes performed on the default log verbosity levels and threshold can lead to an increase in storage size, which might make your database unavailable. These potential impacts on the database hence remain your responsibility.
+</Message>
+
+
+### Disk full
+
+Once full, a database will switch to the `disk_full` state and only accept read requests.
+
+To troubleshoot the issue, you can either:
+
+    - Increase disk volume size, or
+    - Clear data from your database.
+
+Refer to the [Dealing with disk_full mode in a Database Instance](/managed-databases/postgresql-and-mysql/troubleshooting/disk-full/) documentation page for more information.
+
+<Message type="note">
+  You are responsible for monitoring disk size growth and performing any required disk size increases.
+</Message>
+
+
+### Database performance
+Scaleway creates your Database Instance with a default configuration fine-tuned to optimize database performance for standard workloads and monitor and solve issues caused by hardware, operating systems, or file system level failures.
+
+You are responsible for:
+    - Any impacts on performance incurred from SQL queries or database-specific configurations you set up, such as creating indexes, triggering backups, or changing advanced settings  (e.g. `max_connections`).
+    - Sizing the Database Instance appropriately for your workload.
+
+## Configuration and version management
+
+Configuration and version management are critical for maintaining consistency, reliability, and scalability across your services, and to ensure that deployments are accurately configured, tracked, and updated.
+
+This practice fosters streamlined development processes, minimizes errors, and facilitates efficient troubleshooting, ultimately enhancing overall system stability and agility.
```

Comment:
```suggestion

```
I would remove this line ("This practice (...) agility"), I find it too generic to provide significant information to a customer